frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineMetricsRefreshDropdown.tsx

```diff
@@ -14,8 +14,16 @@ const PipelineMetricsRefreshDropdown: React.FC<PipelineMetricsRefreshDropdownPro
   const { t } = useTranslation();
   return (
     <div className="form-group">
-      <label>{t('pipelines-plugin~Refresh Interval')}</label>
-      <IntervalDropdown interval={interval} setInterval={setInterval as never} />
+      <label htmlFor="pipeline-refresh-interval-dropdown">
+        {t('pipelines-plugin~Refresh Interval')}
+      </label>
+      <div>
+        <IntervalDropdown
+          id="pipeline-refresh-interval-dropdown"
+          interval={interval}
+          setInterval={setInterval as never}
```

Comment:
nit: Now that the Interval dropdown now has the proper type as part of this [commit](https://github.com/openshift/console/commit/f0ecc898e5c1d996c896fafa248e1baa152ae3e2#diff-bca699142b1afb14195e2d8057741c21d70040724ca6686edf5e0fb9256a4f44R11) , we can drop the type assertion.
```suggestion
          setInterval={setInterval}
```